apps/omg_watcher_info/test/omg_watcher_info/api/block_test.exs

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.API.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.Fixtures
+  use OMG.WatcherInfo.Fixtures
+  use OMG.Watcher.Fixtures
+
+  alias OMG.WatcherInfo.DB
+  alias OMG.WatcherInfo.API.Block
+
+  describe "get_block/1" do
+    @tag fixtures: [:initial_blocks]
```

Comment:
@unnawut please guide @okalouti so that we avoid using fixtures pls.

Reply:
@okalouti ok this first PR is now official as @InoMurko has weighed in. 😁

@InoMurko jokes aside, I'm glad to hear this about fixtures. I've found them really tricky to trace where data is coming from. What's the new recommended way...can you point to a recent example PR? I need to setup similar test data. Thank you!

Reply:
This is the simplest example without a 3rd party test data factory I think: https://github.com/omisego/ewallet/blob/master/apps/ewallet_db/test/ewallet_db/transaction_test.exs#L45-L52:

```elixir
  describe "get/1" do
    test "retrieves a transaction by idempotency token" do
      {:ok, inserted_transaction} = :transaction |> params_for() |> Transaction.get_or_insert()
      transaction = Transaction.get_by_idempotency_token(inserted_transaction.idempotency_token)

      assert transaction.id == inserted_transaction.id
    end
  end
```

It'll also require this test setup to checkout a DB sandbox to completely remove the current fixtures: https://github.com/omisego/ewallet/blob/master/apps/ewallet_db/test/support/schema_case.ex#L67

```elixir
  setup do
    :ok = Sandbox.checkout(EWalletDB.Repo)
  end
```